File: Algorithms/01-Warmup/02-compare-the-triplets/02.cpp
#include <iostream>
#include <vector>

std::vector < int > solve(std::vector<int> a, std::vector<int> b){
    std::vector<int> total = {0, 0};
    for (int i = 0; i < a.size(); i++) {
        if (a[i] > b[i]) {
            total[0] += 1;
        } else if (a[i] < b[i]) {
            total[1] += 1;
        }
    }
    return total;
}

int main() {

    std::vector<int> a = {0, 0, 0};
    std::vector<int> b = {0, 0, 0};
    std::cin >> a[0] >> a[1] >> a[2];
    std::cin >> b[0] >> b[1] >> b[2];

    std::vector<int> result = solve(a, b);

    for (ssize_t i = 0; i < result.size(); i++) {
        std::cout << result[i] << (i != result.size() - 1 ? " " : "");
    }
    std::cout << std::endl;

    return 0;
}

File: Algorithms/01-Warmup/02-compare-the-triplets/02.java
import java.util.Scanner;

class Solution {

    static int[] solve(int[] a, int[] b){
        int[] total = new int[] {0, 0};
        for (int i = 0; i < a.length; i++) {
            if (a[i] > b[i]) {
                total[0] += 1;
            } else if (a[i] < b[i]) {
                total[1] += 1;
            }
        }
        return total;
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int[] a = new int[] {in.nextInt(), in.nextInt(), in.nextInt()};
        int[] b = new int[] {in.nextInt(), in.nextInt(), in.nextInt()};
        int[] result = solve(a, b);
        for (int i = 0; i < result.length; i++) {
            System.out.print(result[i] + (i != result.length - 1 ? " " : ""));
        }
        System.out.println("");
    }
}
